update_api_files.sh

```diff
@@ -6,7 +6,7 @@
 # It will convert the service descriptions of the specified SDK version
 # (see SDK_VERSION_TAG) into Lua modules and generate a rockspec.
 
-SDK_VERSION_TAG=v2.751.0
+SDK_VERSION_TAG=v2.1353.0
```

Comment:
This is a tricky change. We have way too little test coverage over different AWS services to find potential issues.

wdyt @windmgc ?

Reply:
I'm afraid so... I do not have much confidence in upgrading it directly